mne_bids/mne_bids.py

```diff
@@ -136,10 +141,12 @@ def _channels_tsv(raw, fname, overwrite=False, verbose=True):
                       ('name', raw.info['ch_names']),
                       ('type', ch_type),
                       ('units', units),
-                      ('description', description),
                       ('sampling_frequency', np.full((n_channels), sfreq)),
                       ('low_cutoff', np.full((n_channels), low_cutoff)),
                       ('high_cutoff', np.full((n_channels), high_cutoff)),
+                      ('notch', np.full((n_channels), 'n/a')),
```

Comment:
isn't notch in `raw.info` somewhere? Otherwise, it may not be too difficult to infer from peaks/troughs of PSD (shouldn't be a blocker for this PR though)